src/components/Tooltip/Tooltip.st.css

```diff
@@ -0,0 +1,55 @@
+:import {
+  -st-from: "wix-ui-core/dist/src/components/Tooltip/Tooltip.st.css";
+  -st-default: Tooltip;
+}
+
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: tableSelected, main, mainHover, disabled, successNotifications, dangerNotifications, ctaHover, danger, successHover, dangerHover, disabledDividers;
+}
+
+.root {
+  -st-states: bounce, right, left, top, bottom;
+  -st-extends: Tooltip;
+}
+
+.root:bounce::contentWrap {
+  animation-duration: 1.15s;
+  animation-iteration-count: infinite;
+}
+
+.root:bounce:right::contentWrap {
+  animation-name: right-bounce;
+}
+
+.root:bounce:left::contentWrap {
+  animation-name: left-bounce;
+}
+
+.root:bounce:top::contentWrap {
+  animation-name: top-bounce;
+}
+
+.root:bounce:bottom::contentWrap {
+  animation-name: bottom-bounce;
+}
+
+@keyframes right-bounce {
```

Comment:
is this still the right way to animate now?

Reply:
will implement to implement using our mechanism

Reply:
didnt work, classes are only partially applied